examples/a_treeple/treeple_tutorial_MI.py

```diff
@@ -0,0 +1,97 @@
+"""
+===================================
+Treeple tutorial for calculating MI
```

Comment:
Same comment here regarding adding a paragraph describing what will be shown. 

Also spell out MI before using abbreviation.